libs/zopengl/src/wrapper.zig

```diff
@@ -2719,10 +2719,47 @@ pub const INT_2_10_10_10_REV = bindings.INT_2_10_10_10_REV;
 //     normalized: Boolean,
 //     value: [*c]const Uint,
 // ) callconv(.C) void = undefined;
+
 //--------------------------------------------------------------------------------------------------
 //
 // OpenGL ES 1.0
 //
 //--------------------------------------------------------------------------------------------------
 // pub var clearDepthf: *const fn (depth: Float) callconv(.C) void = undefined;
 // pub var depthRangef: *const fn (n: Clampf, f: Clampf) callconv(.C) void = undefined;
+
+//--------------------------------------------------------------------------------------------------
+//
+// OES_vertex_array_object (OpenGL ES Extension #71)
+//
+//--------------------------------------------------------------------------------------------------
+pub const VERTEX_ARRAY_BINDING_OES = bindings.VERTEX_ARRAY_BINDING_OES; // TODO: This is a pname accepted by getBoolean, getFloat and getInteger
+
+// pub var bindVertexArrayOES: *const fn (array: Uint) callconv(.C) void = undefined;
+pub fn bindVertexArrayOES(array: VertexArrayObject) void {
+    bindings.bindVertexArrayOES(@bitCast(Uint, array));
+}
+
+// pub var deleteVertexArraysOES: *const fn (
+//     n: Sizei,
+//     arrays: [*c]const Uint,
+// ) callconv(.C) void = undefined;
+pub fn deleteVertexArrayOES(ptr: *const VertexArrayObject) void {
+    bindings.deleteVertexArraysOES(1, @ptrCast([*c]Uint, ptr));
+}
+pub fn deleteVertexArraysOES(arrays: []const VertexArrayObject) void {
+    bindings.deleteVertexArraysOES(arrays.len, @ptrCast([*c]Uint, arrays.ptr));
+}
+
+// pub var genVertexArraysOES: *const fn (n: Sizei, arrays: [*c]Uint) callconv(.C) void = undefined;
+pub fn genVertexArrayOES(ptr: *VertexArrayObject) void {
+    bindings.genVertexArraysOES(1, @ptrCast([*c]Uint, ptr));
+}
+pub fn genVertexArraysOES(arrays: []VertexArrayObject) void {
+    bindings.genVertexArraysOES(arrays.len, @ptrCast([*c]Uint, arrays.ptr));
+}
+
+// pub var isVertexArrayOES: *const fn (array: Uint) callconv(.C) Boolean = undefined;
+pub fn isVertexArrayOES(array: VertexArrayObject) bool {
+    return bindings.isVertexArrayOES(@bitCast(Uint, array)) == TRUE;
+}
```

Comment:
The extension maps directly to equivalent core profile api. Maybe we could consider a stretch goal for the wrapper to abstract extensions away. It would simplify usage code, for example:
```zig
switch (options.renderer) {
    .gles2 => {
        gl.genVertexArrayOES(&vao);
        gl.bindVertexArrayOES(vao);
    },
    else => {
        gl.genVertexArray(&vao);
        gl.bindVertexArray(vao);
    },
}
gl.vertexAttribPointer(
    self.vertex_attributes.position,
    3,
    .float,
    gl.FALSE,
    @sizeOf(Vertex3d),
    @offsetOf(Vertex3d, "position"),
);
gl.enableVertexAttribArray(self.vertex_attributes.position);

...

switch (options.renderer) {
    .gles2 => gl.bindVertexArrayOES(vao),
    else => gl.bindVertexArray(vao),
}
```